cl/scrapers/management/commands/scrape_caselaw.py

```
@@ -0,0 +1,339 @@
+
+            for result in results:
+                # We go through each result
+                id = result.get("id")
+                filename = f"{id}.json"
+                if type == "cases":
+                    volume_number = (
+                        result.get("volume").get("volume_number")
+                        if result.get("volume")
+                        else None
+                    )
+                    first_page = result.get("first_page")
+                    reporter_id = (
+                        result.get("reporter").get("id")
+                        if result.get("reporter")
+                        else None
+                    )
+                    reporter_url = (
+                        result.get("reporter").get("url")
+                        if result.get("reporter")
+                        else None
+                    )
+
+                    reporter_slug = get_reporter_slug(reporter_url)
+
+                    if not reporter_slug:
+                        # No slug from url, use reporter id
+                        reporter_slug = reporter_id
+
+                    filename = f"{first_page}.{id}.json"
+                    # Specific directory tree for cases
+                    directory = Path(
+                        settings.MEDIA_ROOT,
+                        "harvard_corpus",
+                        f"law.free.cap.{reporter_slug}.{volume_number}",
+                    )
+
+                file_path = Path(directory, filename)
+
+                if os.path.exists(file_path):
+                    logger.info("Already captured: %s", file_path)
+                    continue
+
+                Path(directory).mkdir(parents=True, exist_ok=True)
+
+                with open(file_path, "w") as outfile:
+                    json.dump(result, outfile, indent=2)
+
+        else:
+            # Something happened, we should probably stop and check what's going on.
+            return
```

do we need to throw an error to sentry here or some logging?